server/cmd/bblfsh-web/main.go

```diff
@@ -55,27 +51,12 @@ func main() {
 	r.Use(gin.RecoveryWithWriter(w))
 	r.Use(gin.LoggerWithWriter(w))
 
-	dir, err := mountAssets()
-	if err != nil {
-		logrus.Fatalf("unable to mount assets: %s", err)
-	}
-
-	assets, err := asset.AssetDir("build")
-	if err != nil {
-		logrus.Fatalf("cannot list assets: %s", err)
-	}
-
-	for _, a := range assets {
-		if a != "static" {
-			r.StaticFile("/"+a, filepath.Join(dir, a))
-		}
-	}
-	indexPath := filepath.Join(dir, "index.html")
-	r.StaticFile("/", indexPath)
-	r.Static("/static", filepath.Join(dir, "static"))
 	server.Mount(s, r.Group("/api"))
-	// we handle urls on frontend
-	r.NoRoute(func(c *gin.Context) { c.File(indexPath) })
+
+	// TODO(@smacker): add configuration for ServerURL and FooterHTML
```

Comment:
This is functionality brought from the code c&p from lookout, but we can't use yet in bblfsh web, right?
Should we have an issue for this, to implement it later?

Reply:
https://github.com/bblfsh/web/issues/201